feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewUiState.kt

```diff
@@ -87,6 +88,11 @@ sealed interface StabilizationUiState {
     }
 }
 
+sealed interface LowLightBoostUiState {
+    data object Active : LowLightBoostUiState
+    data object Inactive : LowLightBoostUiState
+}
```

Comment:
I think this could just be encoded within the `FlashModeUiState.Available` class. Instead of creating a separate class for LLB, just adding an `isActive` property to `FlashModeUiState.Available` should be sufficient. You can then determine when to show different UI when the flash mode is active. This works for low-light boost for now, but in the future we can also extend it to the other flash modes (like making the icon yellow when the torch is on, for example).